src/inspector/node_protocol.pdl

```diff
@@ -92,3 +92,15 @@ experimental domain NodeWorker
       # Identifier of a session which sends a message.
       SessionID sessionId
       string message
+
+# Support for inspecting node process state.
+experimental domain NodeRuntime
+  # When domain is enabled `NodeRuntime.waitingForDisconnect` will be issued at the end of node process.
+  # Please see `NodeRuntime.waitingForDisconnect` for more details.
+  command enable
```

Comment:
"enable" seems to generic a verb for this very specific task. Maybe "retainMainContextForDebugger"?

Reply:
Renamed to notifyWhenWaitingForDisconnect.

Reply:
Did you forget to push the change?

Reply:
it looks like something went wrong yesterday, I just pushed updated version.